app/src/main/java/com/p2p/presentation/tuttifrutti/review/TuttiFruttiReviewRoundAdapter.kt

```diff
@@ -0,0 +1,131 @@
+package com.p2p.presentation.tuttifrutti.review
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.core.content.ContextCompat
+import androidx.recyclerview.widget.RecyclerView
+import androidx.viewbinding.ViewBinding
+import com.p2p.R
+import com.p2p.databinding.ViewReviewCategoryItemBinding
+import com.p2p.databinding.ViewReviewWordItemBinding
+import com.p2p.model.tuttifrutti.FinishedRoundInfo
+import com.p2p.model.tuttifrutti.FinishedRoundPointsInfo
+import com.p2p.presentation.tuttifrutti.create.categories.Category
+import com.p2p.utils.isEven
+
+/** The adapter used to show the list of round reviews. */
+class TuttiFruttiReviewRoundAdapter(private val onChangeRoundPoints: (String, String, Int) -> Unit) :
+    RecyclerView.Adapter<TuttiFruttiReviewRoundAdapter.RecyclerViewHolder>() {
+
+    var finishedRoundInfo = listOf<FinishedRoundInfo>()
+    var finishedRoundPointsInfo = listOf<FinishedRoundPointsInfo>()
+        set(value) {
+            field = value
+            notifyDataSetChanged()
+        }
+
+    override fun getItemViewType(position: Int): Int {
+        // If the the modulus number of players plus one is zero, then we get the category title view
+        return position % (finishedRoundInfo.count() + 1)
```

Comment:
Fantástico esto! Estaría bueno devolver una constante o un enum para hacerlo más claro.
Tener 
```
const val TITLE_TYPE = 0
const val WORD_TYPE = 1
```
y acá devolver 
```suggestion
        return if (position % (finishedRoundInfo.count() + 1) == 0) TITLE_TYPE else WORD_TYPE
```

Si no no termina de quedar claro 😅

Reply:
Re, ahi lo cambio 🤗